### ISSUE TITLE
Bug: Multiple root nodes should be allowed

### ISSUE THREAD

**Issue by jessielaf** [prettier/plugin-pug#145]
## Info

| Tool     | Version       |
| -------- | ------------- |
| Plugin   | v1.10.0       |
| Prettier | v2.1.2        |
| Node     | v12.13.1        |
| OS       | win |

## Input

```pug
DocumentTable(v-if="hasRole(roles.COMPANY_EMPLOYEE)" title="documents.plural")
div(v-else)
  v-tabs.mb-4(v-model="tab" grow @change="fetchData")
    v-tab#user-documents {{ $t('documents.your') }}
    v-tab#company-documents {{ $t('documents.company') }}

  Card
    DocumentTable(ref="documentTable" :title="'documents.' + (tab ? 'company' : 'your')" :add="!Boolean(tab)")
```

## Output or Error

```bash
DocumentTable(v-if="hasRole(roles.COMPANY_EMPLOYEE)" title="documents.plural")
  div(v-else)
    v-tabs.mb-4(v-model="tab" grow @change="fetchData")
      v-tab#user-documents {{ $t('documents.your') }}
      v-tab#company-documents {{ $t('documents.company') }}

    Card
      DocumentTable(ref="documentTable" :title="'documents.' + (tab ? 'company' : 'your')" :add="!Boolean(tab)")
```

## Expected Output

```pug
DocumentTable(v-if="hasRole(roles.COMPANY_EMPLOYEE)" title="documents.plural")
div(v-else)
  v-tabs.mb-4(v-model="tab" grow @change="fetchData")
    v-tab#user-documents {{ $t('documents.your') }}
    v-tab#company-documents {{ $t('documents.company') }}

  Card
    DocumentTable(ref="documentTable" :title="'documents.' + (tab ? 'company' : 'your')" :add="!Boolean(tab)")
```

## Additional Context
I think this change #140 broke with an if else as root element. Also in Vue 3 you can have multiple root elements with [fragments](https://v3.vuejs.org/guide/migration/fragments.html#overview). I now fix this with a div as root element but I think it should be able to have multiple roots.

**Comment by Shinigami92** [comment]
Will dig deeper into the problem a little later
Thanks for the bug report

**Comment by Shinigami92** [comment]
This is a very heavily test driven repo :slightly_smiling_face: 

A first test was immediately successful
Then I assumed that you using the new `pugSingleFileComponentIndentation` config option and wrote a test for this also
I observed that it behaves strange if you don't intend it one level by yourself
But doing so, everything seems to be correct
So maybe it is currently more like a **preserve** one-level indentation for SFCs

Is this your problem? Or is it something different?

You should have a look into my current PR-files: https://github.com/prettier/plugin-pug/pull/146/files

**Comment by lephuongbg** [comment]
A related issue to this is that we use pug's `extends` feature. At the moment, this code

```pug
extends 'some/template.pug'
block header
    div Some header
block content
    div Some content
```

is formatted to

```pug
extends 'some/template.pug'
    block header
        div Some header
    block content
        div Some content
```

**Comment by Shinigami92** [comment]
I'm sorry to inform you that I will probably not have time until the weekend to keep working on it

**Comment by Shinigami92** [comment]
Thanks to @ST-DDT as a co-author in a remote session, it only took around 2 hours to fix this :1st_place_medal: 😜  
This will potentially be released in the next few days together with some other bug fixes.

**Comment by Shinigami92** [comment]
This has now been fixed in release `1.10.1`
Thank you for using my plugin and for being part of the pug community 🤗
